Lib/test/test_pdb.py

```diff
@@ -1453,9 +1453,9 @@ def test_issue26053(self):
             quit
         """
         stdout, stderr = self.run_pdb_script(script, commands)
-        output = '\n'.join([x.strip() for x in stdout.splitlines()])
-        self.assertIn("Restarting main.py with arguments:\na b c", output)
-        self.assertIn("Restarting main.py with arguments:\nd e f", output)
+        res = '\n'.join([x.strip() for x in stdout.splitlines()])
+        self.assertRegex(res, "Restarting .* with arguments:\na b c")
+        self.assertRegex(res, "Restarting .* with arguments:\nd e f")
```

Comment:
Can you use script? Maybe you need os.path.basename(), I don't know pdb.

Reply:
what is script?  The important part of the test is the "a b c" and "d e f"

Reply:
The filename comes from Bdb.canonic:

    ```
def canonic(self, filename):
        """Return canonical form of filename.

        For real filenames, the canonical form is a case-normalized (on
        case insensitive filesystems) absolute path.  'Filenames' with
        angle brackets, such as "<stdin>", generated in interactive
        mode, are returned unchanged.
        """
        if filename == "<" + filename[1:-1] + ">":
            return filename
        canonic = self.fncache.get(filename)
        if not canonic:
            canonic = os.path.abspath(filename)
            canonic = os.path.normcase(canonic)
            self.fncache[filename] = canonic
        return canonic
```

Reply:
I changed it to use canonic(). It passes on the mac.

Reply:
Oh, I see what you mean - no, script is the code (contents of the script). The name is main.py, hard-coded in run_pdb_script()